lib/fdiff/utils.py

```diff
@@ -7,7 +7,7 @@
 
 def file_exists(path):
     """Validates file path as existing local file"""
-    return os.path.isfile(path)
+    return os.path.exists(path)
```

Comment:
Recommend keeping `os.path.isfile` here.  The `exists` will return `True` on directory paths and the name of the function suggests that we only want to verify that a path is a file.

Reply:
Okay. The trouble is `os.path.isfile()` returns `False` for things like `/dev/null` or `/dev/stdin`. Those sort of things should be valid for this use case.

Reply:
Will return to this issue when I have a look at the fontTools exception.

Reply:
Added support for this when the git flag is used in b7c258f56655efa1105e000c1df1e3224be0447f